buildSrc/src/test/groovy/com/google/firebase/gradle/plugins/license/LicenseResolverPluginSpec.groovy

```diff
@@ -80,7 +80,7 @@ class LicenseResolverPluginSpec extends Specification {
 
```

Comment:
Wow! So the test actually served a purpose and passess?

Reply:
yep, with one minor capitalization change